src/backend/joanie/tests/payment/test_backend_base.py

```diff
@@ -153,6 +155,16 @@ def test_payment_backend_base_do_on_payment_success(self):
             1,
         )
 
+        # - Billing address has been registered, it should be non-reusable and
+        #   the title should be the same as the owner's full name
+        address = Address.objects.get(**billing_address["address"])
```

Comment:
According to the test failing, should probably be this ?
```suggestion
        address = Address.objects.get(**billing_address)
```